packages/round-manager/src/features/round/ViewRoundResults/ViewRoundResults.tsx

```diff
@@ -539,13 +541,7 @@ function ViewRoundResults({
                       </RadioGroup.Option>
                       <RadioGroup.Option value={false}>
                         {({ checked }) => (
-                          <div
-                            className={classNames(
-                              "cursor-pointer flex my-2",
-                              disableRoundSaturationControls &&
```

Comment:
is this intended?

Reply:
It was added by mistake, so removed it. 
disableRoundSaturationControls is not relevant to this radioButton group.